src/components.js

```diff
@@ -4,15 +4,17 @@ import { ContainerMixin } from './ContainerMixin';
 export const SlickList = {
   name: 'slick-list',
   mixins: [ ContainerMixin ],
+  props: [ 'tag' ],
   render (h) {
-    return h('div', this.$slots.default);
+    return h(this.$props.tag || 'div', this.$slots.default);
```

Comment:
This is a good place for props default syntax
```
{
  tag: {
    type: String,
    default: 'div',
  }
}
```

Reply:
True dat. On it.

Reply:
Thanks for the PR btw! I'm working on v2. Do you think making the defaults `ul` and `li` would be better? It would introduce a breaking change, but I have plenty of those already.